scripts/bundle/umd.js

```diff
@@ -0,0 +1,12 @@
+function umdWrapper(globalName) {
+  const header = `(function (global, factory) {
+    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
```

Comment:
Could we use a little less ternary here 😂  ?

I understand it's just some boilerplate, but... if we ever need to debug this or understand why Headless does not load properly in a specific scenario or environment, this would be hell to step through.